examples/complete-kubernetes-addons/main.tf

```diff
@@ -178,41 +142,6 @@ module "eks_blueprints_kubernetes_addons" {
     ]
   }
 
-  enable_fargate_fluentbit = true
-  fargate_fluentbit_addon_config = {
```

Comment:
this is nearly (99.9%) identical to the defaults so removing entirely. usage of this addon is still shown though in the `fargate-serverless` example